prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala

```diff
@@ -26,19 +28,13 @@ class SampleConfigCreator extends EmptyProcessConfigCreator {
   )
 
   override def services(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[Service]] = {
+    val mlfConfig = MLFConfig()(processObjectDependencies.config)
     val repo = new MLFRepository()
     val response = repo.listModels
 
-    if(response.isRight) {
-      val modelsList = response.right.get
-      modelsList.foldLeft(Map.empty[String, WithCategories[Service]])(
-        (services, model) => services + (model.getName.toString -> allCategories(new PrinzEnricher(model)))
-      )
-    }
-    else {
-      logger.error(s"Unable to download available models: ${response.left.get.toString}")
-      Map()
-    }
+    response.right.map(modelsList => modelsList.foldLeft(Map.empty[String, WithCategories[Service]])(
```

Comment:
Shouldn't we check if this Either object has right value, that is response didn't return error?